### DISCUSSION TITLE
Reworked on the Logo

### DISCUSSION THREAD

**Discussion by XelXen** [libre-tube/LibreTube#5947] [General]
# The new logo is an enhanced iteration of the logo with an overall better shape and colors. The banner style is also updated with an overall cleaner aesthetic

![Logo](https://github.com/libre-tube/LibreTube/assets/78258352/100a12cd-6828-4ff2-b9e8-907aa161df1e)
![Text Banner](https://github.com/libre-tube/LibreTube/assets/78258352/624e65ec-8ae1-4207-bda4-8fdd154fb0b8)
![Logo Banner](https://github.com/libre-tube/LibreTube/assets/78258352/049bf3ed-1555-4ae5-ad5a-5af65d7bf085)

# Neutral

![Dark](https://github.com/libre-tube/LibreTube/assets/78258352/54ce2a88-92e9-4a44-a447-3ce084f8e8f4)
![Light](https://github.com/libre-tube/LibreTube/assets/78258352/0a8828bf-ffce-40e9-86e0-03e5620c95fb)

# Monotone

![Light](https://github.com/libre-tube/LibreTube/assets/78258352/9730c0a9-64a7-4502-b1e6-92d0cd329351)
![Dark](https://github.com/libre-tube/LibreTube/assets/78258352/e52faa44-84cc-4005-bc40-9338e33ca49e)

# Transparent

![White](https://github.com/libre-tube/LibreTube/assets/78258352/2ab5597d-7b2d-4568-bd8c-8dfc67dcc616)
![Black](https://github.com/libre-tube/LibreTube/assets/78258352/cd6adb45-a163-4ad0-9f7c-b513d8d6ea05)

# SVG

![SVG](https://github.com/libre-tube/LibreTube/assets/78258352/785283da-7dec-4bd1-b46d-266c152ec14a)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by XelXen** [comment]
Logo Comparison

![COMPARISON](https://github.com/libre-tube/LibreTube/assets/78258352/748de567-dce0-4a4f-bc8d-24b6edda0778)

**Comment by XelXen** [comment]
@Bnyro hey dude, long time no see! Any opinions on this?

**Comment by Bnyro** [comment]
Hey @XelXen, hope you're doing well!

Thanks for your efforts with that :+1:

I really like the new banners, we can very well replace the old ones with these in my opinion :)

I'm not sure about the icon change though - I pretty much love the current icon, and I sorta am really too lazy to replace the current icons everywhere (website, readme, inside the app, ...). Though I think we can do this in the future (or you can create PRs if you want)

**Comment by XelXen** [comment]
Alright then, I'll be creating a bunch of PRs then 😂

Can you just list me out the places the icons have to be changed and I'll making variants for them :D

**Comment by Bnyro** [comment]
> Can you just list me out the places the icons have to be changed and I'll making variants for them :D
>
- At the website repo (https://github.com/libre-tube/libre-tube.github.io) - you probably have more of an idea of the images located there :)
- via Android Studio via the inbuilt menu `File -> New -> Image Asset` - for the normal and default light icon (so you add the assets with the name `ic_launcher` and `ic_launcher_light`.
- GitHub profile icon of the LibreTube organization and @libretube-bot - I can do that
- Mastodon - maybe @ARBoyGo can do that?
- Lemmy
- Telegram (I don't have access to that anymore, my account no longer exists I believe)
- I probably forgot something

**Reply by ARBoyGo** [reply]
Sure, I can update for Mastodon and Lemmy.. 
Sorry for late response.

**Comment by ARBoyGo** [comment]
@XelXen I love the banners. Super cool Design Nice work. @Bnyro There is two options for banners.. Which one should we consider for further use case? Personally I like both.

**Reply by ARBoyGo** [reply]
@XelXen If you have Masto/Fedi Account please share..

**Reply by ARBoyGo** [reply]
I just realized that in the banner, there will no longer be the slogan "Privacy Simplified". IDK IG it's fine.
